vault/plugin_catalog.go

```diff
@@ -874,42 +869,3 @@ func isPluginType(s string) bool {
 	_, err := consts.ParsePluginType(s)
 	return err == nil
 }
-
-func (c *PluginCatalog) getBuiltinVersion(pluginType consts.PluginType, pluginName string) string {
```

Comment:
I moved this function to more central place so it could be used more easily. It didn't really need access to the `pluginCatalog` to work.